isis/src/mro/apps/hijitreg/Instrument.cpp

```diff
@@ -414,7 +415,7 @@ namespace UA {
       return
         (int)rint(CCD_FOCAL_PLANE_X_OFFSETS_MM[CPMM]
                   / (CCD_PIXEL_SIZE_MM * binning));
-    }
```

Comment:
For this, we were using a dynamic throw block that would throw either of the 2 exceptions in the block itself. 
These dynamic throws are deprecated and removed from c++17 onward, so I moved the individual throws to be outside the throw() block and got rid of it. If anyone see's an issue with this please let me know.

Reply:
Confirmed that this should not be an issue.